package.json

```diff
@@ -46,8 +46,8 @@
     "lodash": "^4.0.0"
   },
   "peerDependencies": {
-    "graphql": "^0.5.0 || ^0.6.0 || ^0.7.0 || ^0.8.0 || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14",
-    "graphql-relay": "^0.4.2 || ^0.5.0 || ^0.6.0",
+    "graphql": "^0.5.0 || ^0.6.0 || ^0.7.0 || ^0.8.0 || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14 || ^15 || ^16",
+    "graphql-relay": "^0.4.2 || ^0.5.0 || ^0.7.0 || ^0.8.0 || ^0.9.0 || ^0.10.0",
     "sequelize": ">=3.0.0"
```

Comment:
Does this need to be bumped also?

Reply:
yep, this is one of the main complaints I get when I try to update packages, because 0.10.0 has a peerDep on 16+ https://github.com/graphql/graphql-relay-js/blob/main/package.json#L43

For 16 you need 0.10 and the lower ones still work as well

Reply:
yep, apart from the tests now running in v6, all I really added for sequelize was the findByPk in the ternary and fallback to constructor.options